### ISSUE TITLE
Error creating FDW

### ISSUE THREAD

**Issue by decibel** [daamien/fdw_fdw#2]
When trying to install from current source:

``` sql
psql (9.5.0)
Type "help" for help.

~@decina.local/96081# CREATE EXTENSION multicorn;
CREATE EXTENSION
~@decina.local/96081# CREATE SERVER fdw_fdw 
FOREIGN DATA WRAPPER multicorn
options (
  wrapper 'fdwfdw.FDWFDW'
);
ERROR:  Error in python: ImportError
DETAIL:  No module named request
```

This is after installing multicorn via pgxn, which gave me version 1.3.1. I'm using Python 2.7.10 on OS X.

**Comment by daamien** [comment]
Yes that's because I just switched to python 3 and the current version is not compatible with python 2 anymore. Sorry about that.

You have 3 possibilites:

1- Install python3 on your machine

2- check out the python2 branch of this repo and install it 
https://github.com/daamien/fdw_fdw/tree/python2
https://github.com/daamien/fdw_fdw/archive/0.1.0.zip

3- use the docker image
